framerate: 5

camera_image_topic: /camera_pub/image_rect
camera_info_topic: /camera_pub/camera_info

capture_width:  1280
capture_height: 720

image_pub_width: 640
image_pub_height: 360

cam_K: [673.2482675259557, 0.0, 329.80724044927996, 0.0, 675.6751413943377, 180.66056825004506, 0.0, 0.0, 1.0]
cam_D: [0.21887280107995002, -0.4072944753663401, -0.00013698773470533307, 0.0009668096667496951, 0.0]
cam_R: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
cam_P: [693.0473022460938, 0.0, 330.2610844036044, 0.0, 0.0, 696.1262817382812, 180.62339938821424, 0.0, 0.0, 0.0, 1.0, 0.0]